--- xml_data_handler-1.0.4/get_xml_data.h.orig	2014-04-24 19:12:36.000000000 +0200
+++ xml_data_handler-1.0.4/get_xml_data.h	2014-05-15 17:24:27.982275257 +0200
@@ -34,7 +34,7 @@
 }
 
 namespace xml_data {
-    void get_data_values_as_xml(libdap::DataDDS *dds, XMLWriter *writer);
+    void get_data_values_as_xml(libdap::DataDDS *dds, libdap::XMLWriter *writer);
     libdap::DataDDS *datadds_to_xd_datadds(libdap::DataDDS *dds);
     libdap::BaseType *basetype_to_xd(libdap::BaseType *bt);
 }
